### DISCUSSION TITLE
Hartk mod support for Z

### DISCUSSION THREAD

**Discussion by Ixian** [dustin-range/3D_Printer#3] [General]
Hi there!

Thanks again for these projects - I have mine mounted and ready. I was wondering if you'd consider supporting the alternate Z-stop by Hartk aka the "Voron sexbolt" mod? This one: https://mods.vorondesign.com/detail/t1DBVlcUBbdEK6habEsVzg

The hole pattern is slightly different, other than that it looks like it would work. I've struggled a bit with the CAD but my Fusion 3d skills are pretty basic. 

Thanks again for the mods! I'm using the brush/scrub one too.

**Comment by dustin-range** [comment]
Glad to hear you're using these!

Which endstop mount are you using? (Ender/Prusa/GCR/GCR-mirror)

That's a bummer that the SexBolt is just a little different. I loaded the CAD and I see what you mean. It's different just enough that this will be a different version. For example Ender will have a standard and sexbolt. Same with any mount system that get support for it.

I think I can get this done. I prefer to make the change to the mount you need, instead of all 4. In the future if more people need it, I might consider adding it to those.

**Reply by Ixian** [reply]
Thanks! I am using the Ender version (built an Enderwire, Darkdog's Rev2). Will the hole pattern for the mount itself stay the same? I've already drilled the bed carriage and attached it.

**Comment by dustin-range** [comment]
Here ya go!
https://github.com/treeific/3D_Printer/blob/main/z-stop/STLs/z-stop-ender-mount%20(sexbolt).stl

I have not tested it so let me know if you run into problems

**Reply by Ixian** [reply]
Hey, thanks so much! I've printed and tested it; it looks like it would work however it is shifted to the left 3mm - see attached pic. 
![IMG_2520](https://github.com/treeific/3D_Printer/assets/1704047/943fb0ea-011b-426d-9eb1-f35be41e0dc8)

**Comment by Ixian** [comment]
Do you have the CAD for this available? I'm happy to adjust it in Fusion so it lines up.

**Comment by dustin-range** [comment]
I provided the CAD.

https://github.com/treeific/3D_Printer/blob/main/z-stop/CAD/Nozzle%20Probe%20and%20Decontaminator%20v50.f3z

You're welcome to make any adjustments.
